### DISCUSSION TITLE
Would the multichannel module assist with stereo FM?

### DISCUSSION THREAD

**Discussion by ghost** [jgaeddert/liquid-dsp#324] [How do I....?]
I need to simulate stereo FM broadcast activity, I was planning to manually multiplex the two channels prior to using the freqmod module, but the partial documentation on the multichannel module sounded like it could potentially help with the process, is this correct?

**Comment by jgaeddert** [comment]
Right. You're referring to FM broadcast having a primary "mono" L+R in the first 16 kHz or so of the FM broadcast and the stereo double side-band L-R centered at 38 kHz?

**Reply by ghost** [reply]
Yes, that is what I mean :)

**Reply by jgaeddert** [reply]
Short answer is yes.

The long answer is that it might not be necessary. If you first convert to complex using the [firhilb](https://liquidsdr.org/doc/firhilb/) object, you can then slice and dice the spectrum using a channelizer as needed. This would allow you to get all the other carrier data (RDS, for example) pretty easily. But if all you care about is the stereo L-R channel at 38 kHz, it might not be necessary.
